polaris-service/src/main/java/org/apache/polaris/service/task/TableCleanupTaskHandler.java

```diff
@@ -49,6 +51,7 @@ public class TableCleanupTaskHandler implements TaskHandler {
   private final TaskExecutor taskExecutor;
   private final MetaStoreManagerFactory metaStoreManagerFactory;
   private final Function<TaskEntity, FileIO> fileIOSupplier;
+  private static final int BATCH_SIZE = 10;
```

Comment:
Tentatively set the BATCH_SIZE as 10, please feel free to let me know if there is a better option